tests/python/unittest/test_target_codegen_cuda.py

```diff
@@ -511,8 +511,8 @@ def check(t0, t1):
 
         # schedule
         s = tvm.te.create_schedule(C.op)
-        ob, ib = s[C].split(s[C].op.axis[0], nparts=32)
-        _, iib = s[C].split(ib, factor=4)
+        ob, ib = s[C].split(s[C].op.axis[0], nparts=n // factor)
```

Comment:
We can also directly say `factor=factor` here.

Reply:
like this?
```python
ob, ib = s[C].split(s[C].op.axis[0], factor=factor)
# _, iib = s[C].split(ib, factor=factor)
s[C].vectorize(ib)
```

Reply:
Yeah.

Reply:
changed